cmd/local/cmd.go

```diff
@@ -0,0 +1,149 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/kyma-incubator/reconciler/internal/cli"
+	"github.com/kyma-incubator/reconciler/pkg/cluster"
+	file "github.com/kyma-incubator/reconciler/pkg/files"
+	"github.com/kyma-incubator/reconciler/pkg/keb"
+	"github.com/kyma-incubator/reconciler/pkg/logger"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler"
+	"github.com/pkg/errors"
+
+	//Register all reconcilers
+	_ "github.com/kyma-incubator/reconciler/pkg/reconciler/instances"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/workspace"
+	"github.com/kyma-incubator/reconciler/pkg/scheduler"
+	"github.com/spf13/cobra"
+)
+
+const (
+	workspaceDir = ".workspace"
+)
+
+type Options struct {
```

Comment:
to stay with the pattern of the other cmds, it would be good to move all `Options` related code into a separate `options.go` file

Reply:
Done.